apps/ccd/ithc/base/kustomization.yaml

```
@@ -9,6 +9,8 @@ resources:
   - ../../message-publisher/message-publisher.yaml
   - ../../ccd-case-disposer/ccd-case-disposer.yaml
   - ../../../base/slack-provider/ithc
+  - ../../ccd-logstash-civil/ccd-logstash-civil.yaml
```

```suggestion
  - ../../ccd-logstash-civil/ithc.yaml
```
Because you've already applied this ccd-logstash-civil.yaml patch in apps/ccd/ccd-logstash-civil/kustomization.yaml which applies to all